src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestHashMap", "TestHashMap\TestHashMap.csproj", "{EBDC072A-4291-4482-9BAE-2490C190B1B6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateUintCapacity", "HashMap.Test\HashMapNamespace.CalculateUintCapacity\HashMapNamespace.CalculateUintCapacity.csproj", "{4DC0AECB-322A-427A-B49B-FA54AF5A8D16}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateCapacity", "HashMap.Test\HashMapNamespace.CalculateCapacity\HashMapNamespace.CalculateCapacity.csproj", "{52CF5877-0FA0-482F-A9E8-137C1FF052EC}"
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ProvideCapacity", "HashMap.Test\HashMapNamespace.ProvideCapacity\HashMapNamespace.ProvideCapacity.csproj", "{6B3046FE-22DE-43B3-9335-83883D159BA1}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Add", "HashMap.Test\HashMapNamespace.Add\HashMapNamespace.Add.csproj", "{8513B597-4E9C-44CD-809F-290F446E21E4}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.AddOrUpdate", "HashMap.Test\HashMapNamespace.AddOrUpdate\HashMapNamespace.AddOrUpdate.csproj", "{DD451542-3BE6-47B2-8D29-B614A0EE2F6B}"
```

Comment:
### Compilation Error
```
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj (in 428 ms).
  1 of 2 projects are up-to-date for restore.
  HashMap -> /workspaces/CodeExamples/src/HashMap/bin/Debug/net6.0/HashMap.dll
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(100,36): error CS1501: No overload for method 'AddOrUpdate' takes 3 arguments [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(108,83): warning CS8625: Cannot convert null literal to non-nullable reference type. [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(128,36): error CS1501: No overload for method 'AddOrUpdate' takes 3 arguments [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(102,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(102,39): warning NUnit2007: The actual value should not be a constant - perhaps the actual value and the expected value have switched places (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2007.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(120,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(120,45): warning NUnit2007: The actual value should not be a constant - perhaps the actual value and the expected value have switched places (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2007.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(130,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/AddOrUpdate.cs(130,39): warning NUnit2007: The actual value should not be a constant - perhaps the actual value and the expected value have switched places (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2007.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate/HashMapNamespace.AddOrUpdate.csproj]
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.AddOrUpdate (roost-1709036480) $ 
```